docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -405,6 +405,34 @@ When you are ready to migrate your eZ Publish XmlText content to the eZ Platform
 
 `php -d memory_limit=1536M bin/console ezxmltext:convert-to-richtext --export-dir=ezxmltext-export --export-dir-filter=notice,warning,error --concurrency 4 -v`
 
+**Custom tags and attributes**
+
+eZ Platform now supports custom tags, including inline custom tags, and limited use of custom tag attributes.
+After migrating to RichText, you need to adapt your custom tag config for eZ Platform and rewrite the custom tags in Twig.
+See [Custom tag documentation](../guide/extending/extending_online_editor.md#custom-tags) for more info.
+
+If you configured custom attributes in legacy in OE using [ezoe_attributes.ini](https://github.com/ezsystems/ezpublish-legacy/blob/master/extension/ezoe/settings/ezoe_attributes.ini#L33-L48), note that not all types are supported.
+
+Below is a table of the tags that are currently supported, and their corresponding names in eZ Platform.
+
+| [XmlText](https://github.com/ezsystems/ezpublish-legacy/blob/2019.03/extension/ezoe/settings/ezoe_attributes.ini#L33-L48) | [RichText](https://github.com/ezsystems/ezplatform-richtext/blob/v1.1.3/src/bundle/DependencyInjection/Configuration.php#L17) | Note  |
+| ------------- | ------------- | ----- |
+| `link`        | [`link`](../guide/extending/extending_online_editor.md#example-link-tag) |  Treated as a string, no special UI provided by the eZ Platform editor |
```

Comment:
@alongosz This is still accurate right? after https://github.com/ezsystems/ezplatform-richtext/commit/7a4e142dbe68e1cd05741dcc2e727b7df9b28d95#diff-65823235e931ab95eb4d964db3cb038e

Reply:
I believe it is. AFAIK, no special GUI for this is yet made. @alongosz  please confirm

Reply:
> @alongosz This is still accurate right? after [ezsystems/ezplatform-richtext@7a4e142#diff-65823235e931ab95eb4d964db3cb038e](https://github.com/ezsystems/ezplatform-richtext/commit/7a4e142dbe68e1cd05741dcc2e727b7df9b28d95#diff-65823235e931ab95eb4d964db3cb038e)



> I believe it is. AFAIK, no special GUI for this is yet made. @alongosz please confirm

Currently yes, work in progress via ezsystems/ezplatform-richtext#112

Reply:
Thanks for clarification 👍

But then it seems to be just a matter of time before this is wrong, so maybe rather:
A. (preferred) skip the comment as it will change, and it's not critical to explain the feature here (?)
B. Adjust to something like `Currently treaded as string by UI, until EZP-31307 has been resolved.`

@DominikaK up to you here maybe

Reply:
Removed in the spirit of not writing about what will happen in future.